build.gradle

```diff
@@ -33,7 +33,7 @@ dependencies {
         exclude group: 'org.slf4j'
     }
     compile 'com.google.guava:guava:21.0'
-    compile ('io.protostuff:protostuff-parser:2.2.26') {
+    compile ('io.protostuff:protostuff-parser:0-SNAPSHOT') {
```

Comment:
I'll wait until this gets released to fix this.

Reply:
sure